docs/collections/documents/tutorials/working-with-documents.md

```diff
@@ -140,8 +140,8 @@ Let's assume your
     else:
       pp.pprint(docs)
 
-    # Step5: Blacklist IP Address
-    print(f"5. BLACKLIST the IP...from region: {GLOBAL_URL}, ip: {IP_ADDRESS}")
+    # Step5: Blocklist IP Address
```

Comment:
In all cases, please put a space between the word "step" and the number of the step.

Reply:
Done. Thanks for all the suggestions @diana-macrometa!